cmd/reset/reset-consumer-group-offset.go

```diff
@@ -0,0 +1,28 @@
+package reset
+
+import (
+	"github.com/deviceinsight/kafkactl/operations/consumergroupoffsets"
+	"github.com/spf13/cobra"
+)
+
+var offsetFlags consumergroupoffsets.ResetConsumerGroupOffsetFlags
+
+var cmdResetOffset = &cobra.Command{
+	Use:     "consumer-group-offset GROUP",
+	Aliases: []string{"cgo", "offset"},
+	Short:   "reset a consumer group offset",
+	Args:    cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		(&consumergroupoffsets.ConsumerGroupOffsetOperation{}).ResetConsumerGroupOffset(offsetFlags, args[0])
+	},
+}
+
+func init() {
+	cmdResetOffset.Flags().BoolVarP(&offsetFlags.OldestOffset, "oldest", "", false, "set the offset to oldest offset (for all partitions or the specified partition)")
+	cmdResetOffset.Flags().BoolVarP(&offsetFlags.NewestOffset, "newest", "", false, "set the offset to newest offset (for all partitions or the specified partition)")
+	cmdResetOffset.Flags().Int64VarP(&offsetFlags.Offset, "offset", "", -1, "set initial offset for a partition")
```

Comment:
The help for `kafkactl reset consumer-group-offset` looks as follows for this parameter:
```
  --offset int        set initial offset for a partition (default -1)
```
Is there a reason for calling it the "**initial** offset" here but not in `oldest` and `newest`?

The default value doesn't make sense here, since we get this error when using it: `either offset,oldest,newest parameter needs to be specified`

Reply:
the default could not be removed easily. so I added it to the description for now